sdk/storage/azure-storage-files-datalake/swagger/README.md

```diff
@@ -122,10 +120,20 @@ directive:
 
 ```yaml
 directive:
+  - from: swagger-document
+    where: $["x-ms-paths"].*.*.responses.*.headers
+    transform: >
+      for (const h in $) {
+        if (h === "x-ms-encryption-key-sha256") {
+          $[h]["format"] = "byte";
+        }
+      }
   - from: swagger-document
     where: $.parameters
     transform: >
       $.Continuation["x-ms-client-name"] = "ContinuationToken";
+      $.ListBlobsInclude.items["x-ms-enum"]["name"] = "ListBlobsIncludeFlags";
```

Comment:
This should not stay in `Global Changes` section.

Reply:
move it to new "ListBlobsByHierarchy" part.